main.nf

```diff
@@ -140,6 +142,13 @@ process fastq_status {
     single_end = fq[1] == null ? true : false
     qin = sample_type.startsWith('assembly') ? 'qin=33' : 'qin=auto'
     template(task.ext.template)
+
+    stub:
+    """
+    touch *-error.txt
+    touch fastqs/${sample}*.fastq.gz
+    touch ${task.process}/*
```

Comment:
This might not work since first you need to create a folder via `mkdir` and _then_ you need to create sample files.

Try running the following to see what happens 

```bash
touch FASTQC/*

```

Reply:
But i created fastqc with other stub, should i use mkdir folder for each process? if 2 process use the same folder?

Reply:
Good question. Well, you need to remember that NF executes all processes into separate work directories, therefore there is no chance for the folders to clash. :)